src/xdocs/config_filters.xml

```diff
@@ -991,6 +991,215 @@ public class InputTest {
+                    is specified.</td>
+                <td><a href="property_types.html#regexp">Regular Expression</a></td>
+                <td><code>null</code></td>
+                <td>8.16</td>
+            </tr>
+            <tr>
+                <td>message</td>
+                <td>a Regular Expression matched against the message of the check associated
+                    with an audit event. Optional as long as checks or id is specified.</td>
+                <td><a href="property_types.html#regexp">Regular Expression</a></td>
+                <td><code>null</code></td>
+                <td>8.16</td>
+            </tr>
+            <tr>
+                <td>id</td>
+                <td>a string matched against the ID of the check associated with an audit event.
+                    Optional as long as checks or message is specified.</td>
+                <td><a href="property_types.html#string">String</a></td>
+                <td><code>null</code></td>
+                <td>8.16</td>
+            </tr>
+            <tr>
+                <td>query</td>
+                <td>a string xpath query. It is optional.</td>
+                <td><a href="property_types.html#string">String</a></td>
+                <td><code>null</code></td>
+                <td>8.16</td>
+            </tr>
+        </table>
+      </subsection>
+      <subsection name="Examples" id="SuppressionXpathSingleFilter_Examples">
+        <p>
+          To configure to reject CyclomaticComplexity errors
+          for all methods with name sayHelloWorld
+          inside FileOne ane FileTwo classes:
+        </p>
+        <pre>
+&lt;module name="SuppressionXpathSingleFilter"&gt;
+  &lt;property name="files" value="FileOne.java, FileTwo.java"/&gt;
+  &lt;property name="checks" value="CyclomaticComplexity"/&gt;
+  &lt;property name="query" value="//METHOD_DEF[@text='sayHelloWorld']"/&gt;
+&lt;/module&gt;
+        </pre>
+        <p>
+          To suppress checks for package definitions:
+        </p>
+        <pre>
+&lt;module name="SuppressionXpathSingleFilter&gt;
+  &lt;property name="checks" value=".*"/&gt;
+  &lt;property name="query" query="/PACKAGE_DEF"/&gt;
+&lt;/module&gt;
+        </pre>
+        <p>
+          Suppress checks for parent element of the first variable definition:
+        </p>
+        <pre>
+&lt;module name="SuppressionXpathSingleFilter"&gt;
+  &lt;property name="checks" value=".*"/&gt;
+  &lt;property name="query" value="(//VARIABLE_DEF)[1]/.."/&gt;
+&lt;/module&gt;
+        </pre>
+        <p>
+          Suppress checks for elements which are either class definitions,
+          either method definitions:
+        </p>
+        <pre>
+&lt;module name="SuppressionXpathSingleFilter"&gt;
+  &lt;property name="checks" value=".*"/&gt;
+  &lt;property name="query" value="//CLASS_DEF | //METHOD_DEF"/&gt;
+&lt;/module&gt;
+        </pre>
+        <p>
+          Suppress checks for certain methods:
```

Comment:
please add method names used in example in your wording. It helps create a better connection between text and code.